libsql-server/src/connection/libsql.rs

```diff
@@ -76,60 +60,62 @@ where
     ) -> Result<Self> {
         let txn_timeout = config_store.get().txn_timeout.unwrap_or(TXN_TIMEOUT);
 
-        let mut this = Self {
-            db_path,
-            stats,
-            broadcaster,
-            config_store,
-            extensions,
-            max_response_size,
-            max_total_response_size,
-            auto_checkpoint,
-            current_frame_no_receiver,
-            _db: None,
-            wal_wrapper,
-            encryption_config,
-            block_writes,
-            resolve_attach_path,
-            connection_manager: ConnectionManager::new(txn_timeout),
-            make_wal_manager,
-        };
-
-        let db = this.try_create_db().await?;
-        this._db = Some(db);
-
-        Ok(this)
-    }
+        let (snd, recv) = async_channel::bounded(16);
+        tokio::spawn(async move {
+            let inner = MakeLibSqlConnInner {
+                db_path,
+                stats,
+                broadcaster,
+                config_store,
+                extensions,
+                max_response_size,
+                max_total_response_size,
+                auto_checkpoint,
+                current_frame_no_receiver,
+                wal_wrapper,
+                encryption_config,
+                block_writes,
+                resolve_attach_path,
+                connection_manager: ConnectionManager::new(txn_timeout),
+                make_wal_manager,
+            };
 
-    /// Tries to create a database, retrying if the database is busy.
-    async fn try_create_db(&self) -> Result<LibSqlConnection<W>> {
```

Comment:
Why do we no longer need these retries?

Reply:
It's some legacy stuff, trying to see if that works now, and I'd rather pull connection out of the sender until they stop erroring out